x-pack/plugins/security_solution/scripts/run_cypress/parallel_serverless.ts

```diff
@@ -325,9 +332,30 @@ function waitForKibanaLogin(kbUrl: string, credentials: Credentials): Promise<vo
   return pRetry(fetchLoginStatusAttempt, retryOptions);
 }
 
+const getOverridedProductTypes = (
+  projectConfigurationParameters: ProjectConfigurationParameters
+): ProductType[] => {
+  let productTypes: ProductType[] = [...DEFAULT_CONFIGURATION];
+
+  if (projectConfigurationParameters.tier) {
+    productTypes = productTypes.map((product) => ({
+      ...product,
+      product_tier: projectConfigurationParameters.tier,
+    }));
+  }
+  if (projectConfigurationParameters.endpointAddon && !projectConfigurationParameters.cloudAddon) {
+    productTypes = productTypes.filter((product) => product.product_line !== 'cloud');
+  }
+  if (projectConfigurationParameters.cloudAddon && !projectConfigurationParameters.endpointAddon) {
+    productTypes = productTypes.filter((product) => product.product_line !== 'endpoint');
+  }
```

Comment:
We need to check only one flag in each conditional, if the two addons are disabled they both will need to be filtered out.
```suggestion
  if (!projectConfigurationParameters.cloudAddon) {
    productTypes = productTypes.filter((product) => product.product_line !== 'cloud');
  }
  if (!projectConfigurationParameters.endpointAddon) {
    productTypes = productTypes.filter((product) => product.product_line !== 'endpoint');
  }
```